src/Progress Bar/main.vala

```diff
@@ -0,0 +1,65 @@
+#! /usr/bin/env -S vala workbench.vala --pkg libadwaita-1
+
+public void main () {
+    var first_bar = (Gtk.ProgressBar) workbench.builder.get_object ("first");
+    var second_bar = (Gtk.ProgressBar) workbench.builder.get_object ("second");
+    var play = (Gtk.Button) workbench.builder.get_object ("play");
+    var progress_tracker = (Gtk.Label) workbench.builder.get_object ("progress_tracker");
+
+    var target = new Adw.PropertyAnimationTarget (first_bar, "fraction");
+
+    var animation = new Adw.TimedAnimation (
+                                            first_bar,
```

Comment:
Please add comments to indicate what is each of the parameters.

Reply:
Do you think these comments are okay?